mtp_prisoner_location_admin/apps/prisoner_location_admin/forms.py

```diff
@@ -19,6 +19,10 @@ def __init__(self, *args, **kwargs):
 
     def clean_location_file(self):
         locations = []
+
+        if not self.cleaned_data['location_file'].name.endswith('.csv'):
+            raise forms.ValidationError("Uploaded file must be of CSV type")
```

Comment:
Whoops, forgot the _().

As far as the phrasing goes, I was trying to avoid mentioning the fact that it actually just checks the extension. It is certainly a little clunky though.

Just "Uploaded file must be a CSV"?